src/wrapper.rs

```diff
@@ -329,144 +329,156 @@ macro_rules! glib_wrapper {
     // Object, no class struct, no parents or interfaces
     (
         $(#[$attr:meta])*
-        pub struct $name:ident(Object<$ffi_name:ty, $rust_class_name:ident>);
+        pub struct $name:ident(Object<$ffi_name:ty $(, $rust_class_name:ident)?>);
 
         match fn {
             get_type => || $get_type_expr:expr,
         }
     ) => {
-        $crate::glib_object_wrapper!(@object [$($attr)*] $name, $ffi_name, ::std::os::raw::c_void, $rust_class_name, @get_type $get_type_expr, @extends [], @implements []);
+        $crate::glib_object_wrapper!(@object [$($attr)*] $name, $ffi_name, ::std::os::raw::c_void, @get_type $get_type_expr, @extends [], @implements []);
+        $( pub type $rust_class_name = $crate::object::Class<$name>; )?
```

Comment:
I assume we can omit that now and just go everywhere via `Class`? I.e. leave that additional parameter of the macro unused now but leave it there so everything continues to compile for now until the code generator is updated.

Reply:
Then code that actually relies on the class type existing will break (until it's updated to use `Class<_>`. For instance, `ObjectClass` is referenced in a couple places.

Reply:
Well let's update those at the same time then :)

Reply:
Will be easier on https://github.com/gtk-rs/gtk-rs as that has all repos in one place